### ISSUE TITLE
Consider supporting Xml and Yaml

### ISSUE THREAD

**Issue by marcospassos** [V-labs/VlabsMediaBundle#17]
Add support to all mappings formats would be nice. Are there any implication to doing this?

**Comment by choomz** [comment]
Yes you are right. I'm tagging it as an enhancement, but as I said, actuallly I don't have time to do it :(

PRs are open to handle it. If no PR I will do it when i'll be free (don't know when actually ...).

**Comment by marcospassos** [comment]
No problem. This bundle is very useful for us, as soon as I've time I'll submit a PR. Using [DMR](https://github.com/marcospassos/DMR) is easy as pie.
